device_calendar/lib/src/common/constants.dart

```diff
@@ -2,5 +2,5 @@ class Constants {
   static const String fromJsonMapIsNull = 'The json object is null';
 
   static const String invalidArgument = 'Provided arguments are invalid';
-  static const String createEventArgumentReuirements = 'To create an event you must provide calendar with an id, event with title, start date and end date';
+  static const String createOrUpdateEventArgumentRequirements = 'To create or update an event you must provide calendar with an id, event with title, start date and end date (where start date must be before end date)';
```

Comment:
do they actually pass a calendar object with it's id as well or they pass a a calendar id? if it's the latter then the text should change to more accurately reflect that

Reply:
Just an id, will change the text

Reply:
Actually, we were passing the entire Calendar object. I changed that to accept only an Id